.eslintrc.yml

```diff
@@ -14,6 +14,7 @@ plugins:
 rules:
   '@typescript-eslint/explicit-function-return-type': off
   '@typescript-eslint/no-explicit-any': off
+  import/extensions: off
```

Comment:
e.g. `import/extensions` force us to write `import parse from './utils/parse.ts'` instead of `import parse from './utils/parse'`